packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -0,0 +1,441 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalSpacer as Spacer,
+	__experimentalItemGroup as ItemGroup,
+	__experimentalHeading as Heading,
+	__experimentalUnitControl as UnitControl,
+	__experimentalParseQuantityAndUnitFromRawValue as parseQuantityAndUnitFromRawValue,
+	__experimentalGrid as Grid,
+	__experimentalDropdownContentWrapper as DropdownContentWrapper,
+	__experimentalUseNavigator as useNavigator,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	Dropdown,
+	DropdownMenu,
+	NavigableMenu,
+	RangeControl,
+	Button,
+	FlexItem,
+	ColorPalette,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+import {
+	Icon,
+	plus,
+	shadow as shadowIcon,
+	lineSolid,
+	settings,
+	moreVertical,
+} from '@wordpress/icons';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import Subtitle from './subtitle';
+import ScreenHeader from './header';
+import { defaultShadow } from './shadows-panel';
+import {
+	getShadowParts,
+	shadowStringToObject,
+	shadowObjectToString,
+	CUSTOM_VALUE_SETTINGS,
+} from './shadow-utils';
+
+const { useGlobalSetting } = unlock( blockEditorPrivateApis );
+
+const menuItems = [
+	{
+		label: __( 'Rename' ),
+		action: 'rename',
+	},
+	{
+		label: __( 'Duplicate' ),
+		action: 'duplicate',
+	},
+	{
+		label: __( 'Delete' ),
+		action: 'delete',
+	},
+];
+
+export default function ShadowsEditPanel() {
+	const {
```

Comment:
Not, a blocker as the actions are not yet implemented but things like Rename, Duplicate, and Delete should only appear on the custom "group" right now they also appear on default and theme.